### PR TITLE
[Feature] Date and time fields improvements

### PR DESCRIPTION
**Pull request by MaksymShokin**
@sofiiakvasnevska

## Issue
https://github.com/Fliplet/fliplet-studio/issues/5813

## Description
Added props to date and time field that will determine which options are selected.

## Screencast
date 1
date 2

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

File: templates/configurations/time.interface.hbs [hunk 1]
```diff
@@ -3,10 +3,45 @@
...
+  </div>
+  <div>
+    <div class="radio radio-icon">
+      <input type="radio" id="loaded" name="loaded" value='loaded' v-model="timeApply">
+      <label for="loaded">
+        <span class="check"><i class="fa fa-circle"></i></span>Use the time when form is loaded
+      </label>
+    </div>
+    <div class="radio radio-icon">
+      <input type="radio" id="submitted" name="submitted" value='submitted' v-model="timeApply">
```
Comment (tonytlwu) [inline templates/configurations/time.interface.hbs:37]:
```suggestion
      <input type="radio" id="submitted" name="submitted" value='submission' v-model="defaultSource">
```

File: templates/configurations/time.interface.hbs [hunk 2]
```diff
@@ -3,10 +3,45 @@
...
+  </div>
+</div>
+
+<div class="form-group">
+  <div>
+    <label class="control-label">When appliyng the current time</label>
+  </div>
+  <div>
+    <div class="radio radio-icon">
+      <input type="radio" id="loaded" name="loaded" value='loaded' v-model="timeApply">
```
Comment (tonytlwu) [inline templates/configurations/time.interface.hbs:31]:
```suggestion
      <input type="radio" id="loaded" name="loaded" value='load' v-model="defaultSource>
```

File: templates/components/time.build.hbs [hunk 3]
```diff
@@ -1,18 +1,25 @@
 <div v-if="description" class="help-block description">\{{ description }}</div>
-<div class="input-group custom-time focus-outline">
-  <div class="input-group-addon">
-    <i class="fa fa-clock-o"></i>
+
+<template v-if="autoFill === 'always' && timeApply === 'submitted' && empty">
```
Comment (tonytlwu) [inline templates/components/time.build.hbs:3]:
```suggestion
<template v-if="autofill === 'always' && timeApply === 'submitted' && empty">
```

File: templates/configurations/time.interface.hbs (hunk 1 above)
Comment (MaksymShokin) [reply templates/configurations/time.interface.hbs]: Fixed

File: templates/components/time.build.hbs (hunk 3 above)
Comment (MaksymShokin) [reply templates/components/time.build.hbs]: Fixed

File: templates/configurations/time.interface.hbs (hunk 2 above)
Comment (MaksymShokin) [reply templates/configurations/time.interface.hbs]: Fixed

**Pull request merged**